src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -410,7 +412,8 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
                 services.AddAuthentication()
                     .AddScriptJwtBearer()
-                    .AddScheme<AuthenticationLevelOptions, CliAuthenticationHandler>(AuthLevelAuthenticationDefaults.AuthenticationScheme, configureOptions: _ => { });
+                    .AddScheme<AuthenticationLevelOptions, CliAuthenticationHandler<AuthenticationLevelOptions>>(AuthLevelAuthenticationDefaults.AuthenticationScheme, configureOptions: _ => { })
```

Comment:
This is how the CLI disables auth requirements via DI? The goal for CLI is to disable auth completely right? This pattern seems fragile - any time we add new auth schemes, we have to add a corresponding line here, as opposed to just setting some single "disable auth" switch, which we used to have, but I just removed because it wasn't being used.

Reply:
If there is no other way, we can always add a test here that enumerates auth schemes from the host and fails if they are not present here.

Reply:
This current implementation is fragile for the CLI, but still better than having that flag :)

I added this additional scheme handler to the CLI for now, but we can change this to handle all of the schemes and/or ensure we register the defaults and use override the the authentication level options. I wanted to get this fixed to make sure we didn't miss this release train, but will work on a more permanent fix.

Reply:
Agreed. I don't want to block this, since you're making the right fix for the existing design. I'm just saying that if your goal is to disable ALL the default auth, it should be a one liner to do that, and we shouldn't have to update the CLI code when we add new auth schemes. In that sense a single simple flag (or a services.DisableAuthentication() call or something) better captures the intent and is more maintainable.

Reply:
Makes sense. Will enhance this to make sure it becomes that simple and doesn't require maintenance. But if you guys are OK, I'll merge this to keep moving forward.